model/src/main/kotlin/ScanResult.kt

```diff
@@ -20,6 +20,7 @@
 package org.ossreviewtoolkit.model
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonInclude
```

Comment:
Commit message nits:
- check line wrapping
- quote `additionalData`
- don't only say it's called `additionalData` now, but also why (to emphasize this is about optional data that cannot be captures in the common data model).
- let's say "*will be used" instead of "can be used", so make clear that there is a real need for this *now*

Reply:
> quote `additionalData`

My fault, with "quote" I meant "add backticks" to get formatting exactly as in my example; but let's keep it as-is now.